hphp/runtime/vm/jit/types.h

```diff
@@ -145,6 +145,11 @@ enum class CodeKind : uint8_t {
    * Helper code that uses scratch registers only.
    */
   Helper,
+
+  /*
+   * Fast Stub ABI can only use platform-specific "assembler" scratch registers
```

Comment:
nit: Period at end of comment.  Also, can you indicate that this is intended for code called by `callfaststub{}`?